resources/js/Components/WikidataToolFooter.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="footer-container">
+        <footer :class="contentClass">
+            <section>
+                <h2 class="h5">
+                    {{ $i18n('wikidata-tool-footer-about-tool', labels.tool) }}
+                </h2>
+                <p v-i18n-html:wikidata-tool-footer-license="[urls.license, labels.license ]"/>
+                <p><wikit-link :href="urls.source">{{ $i18n('wikidata-tool-footer-source') }}</wikit-link></p>
+                <p><wikit-link :href="urls.issues">{{ $i18n('wikidata-tool-footer-issues') }}</wikit-link></p>
+            </section>
+            <section>
+                <h2 class="h5">{{ $i18n('wikidata-tool-footer-about-us') }}</h2>
+                <p>
+                    <wikit-link href="https://foundation.wikimedia.org/wiki/Non-wiki_privacy_policy">
+                        {{ $i18n('wikidata-tool-footer-privacy') }}
+                    </wikit-link>
+                </p>
+                <p>
+                    <wikit-link href="https://www.wikimedia.de/">
+                        {{ $i18n('wikidata-tool-footer-wmde') }}
+                    </wikit-link>
+                </p>
+                <p v-i18n-html:wikidata-tool-footer-team="[
+                    'https://www.wikidata.org/wiki/Wikidata:Contact_the_development_team'
+                ]" />
+            </section>
+            <slot />
+        </footer>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import { Link as WikitLink } from '@wmde/wikit-vue-components';
+
+interface FooterLabels {
+    tool: string;
+    license: string;
+}
+
+interface FooterUrls {
+    license: string;
+    source: string;
+    issues: string;
+}
+
+export default Vue.extend({
+    name: 'WikidataToolFooter',
+    components: {
+        WikitLink,
+    },
+    props: {
+        contentClass: {
+            type: String,
+            default: '',
+        },
+        labels: {
+            type: Object as PropType<FooterLabels>,
+            required: true,
+        },
+        urls: {
+            type: Object as PropType<FooterUrls>,
+            required: true,
+        },
+    }
+});
+</script>
+
+<style lang="scss">
+@import '~@wmde/wikit-tokens/dist/_variables.scss';
+
+.footer-container {
+    background-color: $color-base-90;
+
+    & > footer {
+        margin: auto;
+        padding: $dimension-layout-xsmall;
+        display: flex;
+        flex-direction: column;
+
+        .h5 {
+            margin: 0 0 $dimension-layout-xsmall 0;
+        }
+
+        section {
+            margin-block-end: $dimension-layout-small;
```

Comment:
Great work, just one detail... The total space between the text of the sections and the very end of the website in tablet and desktop in the design is 19.32px. Right now these are rending to a total of 3rem (1.5rem from the `<footer>` and 1.5rem from the sections, which for me renders as 27px + 27px)

Reply:
These are the tokens that were required in the specs. Measuring pixels in Figma is not accurate, as Figma serves more of an approximation of the final design. Either or, pixel values should be relevant here since we are measuring by the relative `rem` unit (which are sensitive to the user's Browser preferences)

Reply:
Yes, but here I am mentioning a different issue, there are duplicate paddings. I think the padding of either the section or the footer has to go.

Reply:
Ah! sorry... I was dazzled by numbers

Reply:
Sorry :D I wanted to be clear and ended up writing a convoluted review :sweat_smile: